metaflow/flowspec.py

```diff
@@ -7,7 +7,7 @@
 from types import FunctionType, MethodType
 from typing import Any, Callable, List, Optional, Tuple
 
-from . import cmd_with_io
+from . import cmd_with_io, parameters
 from .parameters import DelayedEvaluationParameter, Parameter
```

Comment:
`from . import cmd_with_io, parameters` --> can we do this instead:
`from .parameters import DelayedEvaluationParameter, Parameter, parameters`

since we already have the snippet for `from .parameters`

Reply:
Actually, the current approach is fine, otherwise we will need to access it as a global variable inside the class..